com.unity.multiplayer.mlapi/Runtime/Core/NetworkManager.cs

```diff
@@ -29,7 +29,7 @@ namespace MLAPI
     /// The main component of the library
     /// </summary>
     [AddComponentMenu("MLAPI/NetworkManager", -100)]
-    public class NetworkManager : MonoBehaviour, INetworkUpdateSystem
+    public class NetworkManager : MonoBehaviour, INetworkUpdateSystem, IProfilableTransportProvider
```

Comment:
I was wondering if we should have `#if` guards around this implementation or not.
would everything work fine and be useful/utilized in 2019.4 LTS?
I'm asking mainly because we might not support all profiling features in both profiler UI/window and inline profiler events/instruments/counters — right?
(maybe I'm missing something completely here, sorry if I'm being silly :P)

Reply:
We put guards around the old original MLAPI profiler. (with UNITY_2020_2_OR_NEWER)
Additionally If they use an older version of unity the code should simply not render. (The renderer is also guarded around UNITY_2020_2_OR_NEWER)

Im not sure if i answered your question. but essentially we shouldn't need any additional guards. On 2019.4 in the worst case scenario it will compile but just not render because of the guards.


If there is an issue even with my "guess" and using UNITY_2020_2_OR_NEWER, we should address it in a new pr